lib/modules/apostrophe-pieces/lib/api.js

```diff
@@ -423,7 +422,7 @@ module.exports = function(self, options) {
         if (err) {
           return callback(err);
         }
-        return self.apos.docs.deduplicateRescue(req, piece, callback);
+        return self.apos.docs.getManager(self.name).deduplicateRescue(req, piece, callback);
```

Comment:
This is redundant, this module is the manager of its own type, so just say self.deduplicateRescue.